WordPressAuthenticator/Unified Auth/View Related/Get Started/GetStartedViewController.swift

```diff
@@ -734,3 +830,30 @@ extension GetStartedViewController: UITextFieldDelegate {
     }
 
 }
+
+// MARK: - Keyboard Notifications
+
+extension GetStartedViewController {
+    @objc func handleKeyboardWillShow(_ notification: Foundation.Notification) {
+        keyboardWillShow(notification)
+    }
+
+    @objc func handleKeyboardWillHide(_ notification: Foundation.Notification) {
+        keyboardWillHide(notification)
+    }
+}
+
+// MARK: - Button configuration
+
+private extension GetStartedViewController {
+    enum ButtonConfiguration {
+        enum Continue {
+            static let title = WordPressAuthenticator.shared.displayStrings.continueButtonTitle
+            static let accessibilityIdentifier = "Get Started Email Continue Button"
```

Comment:
Nit: For accessibility identifiers, I think we want to follow the naming style for IDs instead:
```suggestion
            static let accessibilityIdentifier = "get-started-email-continue-button"
```

But since the naming is not consistent in this library, this is just a suggestion.

Reply:
I would like to keep using the old name as I only refactored and moved this code here. And, I don't want to break any tests that might be using this.